runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNoteOverlay.java

```diff
@@ -58,14 +61,12 @@ public Dimension render(Graphics2D graphics)
 		}
 
 		final HoveredFriend hovered = plugin.getHoveredFriend();
-		if (hovered == null)
+		if (hovered != null)
```

Comment:
Why you flipped this?

Reply:
See the split diff - the method returns null either way so this is easier to follow.